server/utils.go

```diff
@@ -310,3 +308,27 @@ func getCodeMarkdown(user, repo, repoPath, word, lines string, isTruncated bool)
 	final += "```\n"
 	return final
 }
+
+// getToDoDisplayText returns the text to be displayed in todo listings.
+func getToDoDisplayText(title, url, notifType string) string {
+	baseURL := "https://github.com/"
+
+	owner, repo := parseOwnerAndRepo(url, baseURL)
+	repoURL := fmt.Sprintf("%s%s/%s", baseURL, owner, repo)
+	repoWords := strings.Split(repo, "-")
+	if len(repoWords) != 1 {
```

Comment:
maybe `if len(repo) > 20 && len(repoWords) > 1` - no reason to shorten names that are reasonably short?